modules/dop/dao/issue_state_circulation.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dao
+
+import (
+	"github.com/erda-project/erda/pkg/database/dbengine"
+)
+
+type IssueStateCirculation struct {
+	dbengine.BaseModel
+
+	ProjectID uint64
+	IssueID   uint64
+	StateFrom uint64
+	StateTo   uint64
+	Creator   string
+}
+
+func (IssueStateCirculation) TableName() string {
+	return "dice_issue_state_circulation"
```

Comment:
erda_issue_state_circulation

Reply:
The others are prefixed with `dice` like issue,issue_state and so on

Reply:
done